src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
> hat the rtl transformer will be added to the list of transformers if isRtl is set to true

This is a bit scary. :P What if they want to add transformation after rtl is applied?

> Also, I'm thinking that if we make this a documented API we should eventually deprecate isRtl in favor of them just applying the transformer themselves.

I think some components rely on `isRtl`.